thoth-api/src/graphql/model.rs

```diff
@@ -1748,6 +1763,10 @@ impl MutationRoot {
         // update the work and, if it succeeds, synchronise its children statuses and pub. date
         match work.update(&context.db, &data, &account_id) {
             Ok(w) => {
+                let _ = send_event(&context.redis, EventType::WorkUpdated, &w).await;
```

Comment:
To discuss: do we need both a WorkUpdated and a WorkPublished when publishing a book?